appveyor.yml

```diff
@@ -58,6 +71,10 @@ install:
         https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
+  - ps: |
+      If (($env:PYTHON_VERSION -eq "3.5" -or $env:PYTHON_VERSION -eq "3.4") -and ($env:APPVEYOR_REPO_TAG -ne "true")) {
```

Comment:
ist it possible to have an environment variable e.g. ```SKIP_IF_TAG``` within the matrix instead of checking against each version we want to skip here? Would be easier for adding later other/newer environments ...